mycroft/client/enclosure/mark1/__init__.py

```diff
@@ -164,6 +165,9 @@ def process(self, data):
         if "unit.factory-reset" in data:
             self.bus.emit(Message("speak", {
                 'utterance': mycroft.dialog.get("reset to factory defaults")}))
+            subprocess.call(
+                'rm ~/.config/mycroft/identity/identity2.json',
```

Comment:
Should we be using `xdg.BaseDirectory.save_config_path('mycroft')` here?

Reply:
Yes, yes we should.

Reply:
Code updated according to this.